modules/metrics_prometheus_v1/src/main/scala/com/evolutiongaming/skafka/metrics/KafkaMetricsRegistry.scala

```diff
@@ -0,0 +1,82 @@
+package com.evolutiongaming.skafka.metrics
```

Comment:
There's no good option here but maybe we could consider using a different FQN to avoid collisions. As soon as anyone has both this and the old module on the classpath, they will start colliding by picking random implementations

Reply:
I think so, if we plan to deprecate the old one eventually. It will be a breaking change for downstream consumers then, and we'll have to release a new major version

Reply:
Done. I've only changed the packages and organisation for the new module, to preserve the old ones